client/src/components/StreamField/blocks/StreamBlock.js

```diff
@@ -85,6 +85,35 @@ class StreamBlockMenu extends BaseInsertionControl {
     $(placeholder).replaceWith(dom);
     this.element = dom.get(0);
     this.addButton = dom.find('button');
+
```

Comment:
@SebCorbin it would be great to see a unit test for this behaviour if you can.

client/src/components/StreamField/blocks/StreamBlock.test.js

Reply:
Thanks. Tests have been added.